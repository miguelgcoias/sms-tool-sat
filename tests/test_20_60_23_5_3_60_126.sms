20
8 4 20 3 1 1 2
0 10 10 4 2 4 2 2
2 10 22 3 6 1 3
10 17 27 1 17
7 23 53 3 3 9 11
11 6 17 3 2 2 2
59 15 74 3 6 2 7
6 14 48 4 3 4 6 1
22 4 26 2 1 3
0 4 4 3 1 1 2
11 14 25 5 5 1 2 2 4
31 7 52 4 1 2 2 2
29 9 47 3 5 1 3
42 20 102 4 6 4 3 7
32 18 86 5 3 5 1 4 5
55 18 109 2 9 9
15 6 33 2 3 3
16 19 54 4 4 5 4 6
0 22 22 5 4 6 6 3 3
26 4 38 4 1 1 1 1
0
0
0
0
2 6 4
0
1 4
0
1 3
0
1 10
3 3 10 2
1 2
1 11
5 5 13 10 11 19
2 3 13
1 1
1 4
0
1 9
