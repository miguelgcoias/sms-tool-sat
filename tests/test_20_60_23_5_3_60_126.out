7
1 10 12 16
3 4 11 13
8 33 36 40 46
9 22 23
10 0 1 2
17 18 27
20 26 30 31 32
